File: 0242-valid-anagram/0242-valid-anagram.java
class Solution {
    public boolean isAnagram(String s, String t) {
        int[] arr = new int[26];
        int[] arr2 = new int[26];
        
        for(char ch : s.toCharArray())
            arr[122   - ch] = arr[122   -ch] + 1;
        
                
        for(char ch2 : t.toCharArray())
            arr2[122   -ch2] = arr2[122   -ch2] + 1;
        
        for(int i = 0; i < 26; i++)
        {
            if(arr[i] != arr2[i])
                return false;
        }
        
        return true;
    }
}

File: 0242-valid-anagram/0242-valid-anagram.py
class Solution:
    def isAnagram(self, s: str, t: str) -> bool:
        left = [0] * 26

        for char in s:
            left[ord('z') - ord(char)] += 1
        
        for char in t:
            left[ord('z') - ord(char)] -= 1

        for value in left:
            if(value != 0):
                return False
        return True
        